test/ops/concat.js

```diff
@@ -44,7 +83,132 @@ describe('test concat', function() {
     }
   });
 
-  it('concat 3d', function() {
+  it('concat 2d axis=0 with two inputs', function() {
+    const tensors = [
+      {desc: {type: 'float32', dimensions: [1, 2]}, value: [1, 2]},
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [3, 4, 5, 6]},
+    ];
+    const expected = {axis: 0, shape: [3, 2], value: [1, 2, 3, 4, 5, 6]};
+    testConcatInputs(tensors, expected);
+  });
+
+  it('concat 2d axis=0 with three inputs', function() {
+    const tensors = [
+      {desc: {type: 'float32', dimensions: [1, 2]}, value: [1, 2]},
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [3, 4, 5, 6]},
+      {
+        desc: {type: 'float32', dimensions: [3, 2]},
+        value: [7, 8, 9, 10, 11, 12],
+      },
+    ];
+    const expected = {
+      axis: 0,
+      shape: [6, 2],
+      value: [
+        1, 2, 3, 4, 5, 6,
+        7, 8, 9, 10, 11, 12,
+      ]};
+    testConcatInputs(tensors, expected);
+  });
+
+  it('concat 2d axis=0 with four inputs', function() {
+    const tensors = [
+      {desc: {type: 'float32', dimensions: [1, 2]}, value: [1, 2]},
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [3, 4, 5, 6]},
+      {
+        desc: {type: 'float32', dimensions: [3, 2]},
+        value: [7, 8, 9, 10, 11, 12],
+      },
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [13, 14, 15, 16]},
+    ];
+    const expected = {
+      axis: 0,
+      shape: [8, 2],
+      value: [
+        1, 2, 3, 4, 5, 6,
+        7, 8, 9, 10, 11, 12,
+        13, 14, 15, 16,
+      ]};
+    testConcatInputs(tensors, expected);
+  });
+
+  it('concat 2d axis=0 with five inputs', function() {
+    const tensors = [
+      {desc: {type: 'float32', dimensions: [1, 2]}, value: [1, 2]},
+      {desc: {type: 'float32', dimensions: [2, 2]}, value: [3, 4, 5, 6]},
+      {
```

Comment:
Do you forget formating these lines?

Reply:
Since one line limits 80 characters，@mingmingtasd

Reply:
> Since one line limits 80 characters，@mingmingtasd

OK, got it, just a reminder, thanks! Approved yet.@BruceDai